### ISSUE TITLE
Allow non-semver versions

### ISSUE THREAD

**Issue by binarykitchen** [AndersDJohnson/webpack-babel-env-deps#51]
This error is thrown in my project

```
Error: Invalid version: "207-alpha1-SNAPSHOT"
```

Our version in package.json is sentimental, not in semver.

In error stack trace I can see above error comes from

```
/node_modules/webpack-babel-env-deps/node_modules/read-pkg/index.js:43:36
```

Not sure if this is related to your package or read-pkg or normalize-package-data. Feel free to close if I'm wrong here. Thanks.

**Comment by AndersDJohnson** [comment]
Can confirm. Checking if there's anything we can do.

```
Invalid version: "1-whatever"

      115 | export function getNeedBabel(options) {
      116 |   const pathInPkg = options.path || process.cwd()
    > 117 |   const pkg = readPkgUp.sync({
          |                         ^
      118 |     cwd: pathInPkg
      119 |   })
      120 |

      at Object.fixVersionField (node_modules/normalize-package-data/lib/fixer.js:191:13)
      at node_modules/normalize-package-data/lib/normalize.js:32:38
          at Array.forEach (<anonymous>)
      at normalize (node_modules/normalize-package-data/lib/normalize.js:31:15)
      at Function.Object.<anonymous>.module.exports.sync (node_modules/read-pkg/index.js:43:36)
      at Function.sync (node_modules/read-pkg-up/index.js:23:16)
      at getNeedBabel (src/index.js:117:25)
      at include (src/index.js:163:17)
      at Object.<anonymous> (src/__tests__/index.test.js:287:14)
```

**Comment by AndersDJohnson** [comment]
Thanks @binarykitchen - I dug in and found the same root cause you did, and found your upstream issue https://github.com/npm/normalize-package-data/issues/116.
For now, I'm trying https://github.com/AndersDJohnson/webpack-babel-env-deps/pull/52 as a workaround.

**Comment by AndersDJohnson** [comment]
@binarykitchen Should be published with `1.6.4`, do you want to give it a shot? I'm seeing some possible Babel dependency issues in one of my projects so please let me know if you also run into those and I can work to fix too.
